### ISSUE TITLE
npm-check freezes computer

### ISSUE THREAD

**Issue by alejandroiglesias** [dylang/npm-check#199]
Hi, just wanted to let you know that npm-check just froze my computer almost immediately after running it. I just uninstalled it for good and according to other bug reports, something's going on with resource usage. Hope you can tackle that down but I would suggest adding a warning notice.

**Comment by scippio** [comment]
In my situation it's similiar.. All my memory/swap, CPU and i/o disk are consumed when I run: `npm-check --skip-unused`... And after 2 minutes.. is killed. 
`⠴ Checking npm registries for updated packages.Killed (SIGKILL)`

**Comment by renatoaraujoc** [comment]
I can confirm this issue.

**Comment by elektronik2k5** [comment]
Yup, just tried it too on my Linux machine - freezes everything for a minute or two :(.

**Comment by dhritzkiv** [comment]
Ditto on macOS 10.12.3. Uses up a many GBs of memory, freezing my computer until it completes

**Comment by D4nt4lion** [comment]
Still an issue

**Comment by BogdanBulai** [comment]
Still an issue. Do we need quantum computers to use your package?
